openfecwebapp/templates/partials/navigation.html

```diff
@@ -47,9 +47,15 @@
     <li class="site-nav__item">
       <a href="{{ cms_url }}/calendar/" class="site-nav__link">Calendar</a>
     </li>
+    {% if features.legal %}
+    <li class="site-nav__item">
+      <a href="{{ cms_url }}/legal-resources" class="site-nav__link">Legal resources</a>
```

Comment:
`/legal-resources` is served by the cms, currently just a search box that takes you to /legal/search. @edmullen has a mock-up of that landing page, so more content to land there soon.